library/src/test/java/com/mapbox/android/gestures/AndroidGesturesManagerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.android.gestures;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@RunWith(RobolectricTestRunner.class)
+public class AndroidGesturesManagerTest {
+  private AndroidGesturesManager androidGesturesManager;
+
+  private Set<Integer> set1 = new HashSet<>();
+  private Set<Integer> set2 = new HashSet<>();
+  private Set<Integer> set3 = new HashSet<>();
+  private List<Set<Integer>> mutuallyExclusivesList = new ArrayList<>();
+
+  @Before
+  public void setUp() throws Exception {
+
+    set1.add(AndroidGesturesManager.GESTURE_TYPE_ROTATE);
+    set1.add(AndroidGesturesManager.GESTURE_TYPE_SCROLL);
+
+    set2.add(AndroidGesturesManager.GESTURE_TYPE_ROTATE);
+    set2.add(AndroidGesturesManager.GESTURE_TYPE_SCALE);
+
+    set3.add(AndroidGesturesManager.GESTURE_TYPE_ROTATE);
+    set3.add(AndroidGesturesManager.GESTURE_TYPE_SCALE);
+    //set3.add(AndroidGesturesManager.GESTURE_TYPE_MOVE);
+
+    mutuallyExclusivesList.add(set1);
+    mutuallyExclusivesList.add(set2);
+    mutuallyExclusivesList.add(set3);
+
+    androidGesturesManager =
+      new AndroidGesturesManager(
+        RuntimeEnvironment.application.getApplicationContext(),
+        mutuallyExclusivesList);
+  }
```

Comment:
this setup method will be called for each test found in `AndroidGesturesManagerTest`, this means if you would add more tests to this class the set1,set2,set3 will grow equally to the amount of tests. I think you want to recreate the lists with each `@Before`